getting_started/index.adoc

```diff
@@ -8,26 +8,55 @@
 :toc: macro
 :toc-title:
 
+[[introduction]]
+
+== Introduction
+
+{product-title} is a platform as a service
+(PaaS) offering from Red Hat that brings together Docker and Kubernetes, and
+provides and API to manage these services. {product-title} allows you to create
+and manage containers.
+
+[[why-use-openshift]]
+=== Why should I use OpenShift?
```

Comment:
@bmcelvee Sorry if this came up in a previous comment and I missed: should this heading be title case? I.e., "Why Should I Use OpenShift?"